app/models/user.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  allow_password_change  :boolean          default(FALSE)
+#  confirmation_sent_at   :datetime
+#  confirmation_token     :string
+#  confirmed_at           :datetime
+#  email                  :string
+#  encrypted_password     :string           default(""), not null
+#  gender                 :integer
+#  image                  :string
+#  name                   :string
+#  nickname               :string
+#  provider               :string           default("email"), not null
+#  remember_created_at    :datetime
+#  reset_password_sent_at :datetime
+#  reset_password_token   :string
+#  tokens                 :json
+#  uid                    :string           default(""), not null
+#  unconfirmed_email      :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+# Indexes
+#
+#  index_users_on_confirmation_token    (confirmation_token) UNIQUE
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
+#  index_users_on_uid_and_provider      (uid,provider) UNIQUE
+#
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  include DeviseTokenAuth::Concerns::User
+
+  enum gender: %i[male female other]
```

Comment:
You should actually use a hash for enums https://naturaily.com/blog/ruby-on-rails-enum

The reason is that if you add another gender and you do it like 
```ruby
  enum gender: %i[male female non_binary other]
```

It would all mappings would be wrong since previous `other` would now be mapped to `non_binary`. There's a [rubocop cop](https://github.com/rubocop-hq/rubocop-rails/commit/1464219dd54e4734d70ea3459b8119fa4b511c8b#diff-ec100fd71c3a4e0e2aeb72bc63060b0eef5a0f462ab7d71a9755437de3c0a694R6) for that which we actually did with some guys at RS 😄